pages/accountLists/[accountListId]/reports/financialAccounts/[[...financialAccount]].page.tsx

```diff
@@ -0,0 +1,172 @@
+import Head from 'next/head';
```

Comment:
Is there a reason we're having this one page handle all financial accounts pages instead of having an `index.page.tsx` for the list, `[financialAccountId].page.tsx` for the summary, and `[financialAccountId]/entries.page.tsx` for the entries?

Reply:
It's more of a preference. I'm okay to split it up into 3 pages, but that will be done in another PR, so we can get this live.